app/exchanges/README.md

```diff
@@ -0,0 +1,75 @@
+# Sample Sheet Generator
+
+The sample sheets are defined through configuration in
+[/config/pipelines/](/config/pipelines/)`{pipeline}.yml` for ONT and PacBio.
+
+This configuration is parsed using
+[/app/exchanges/run_csv/](/app/exchanges/run_csv/)`{pipeline}_sample_sheet.rb` as appropriate.
+
+The required format of the sample sheets can be found on page 27 of the [SMRT Link User Guide
```

Comment:
I'd make this less specific as we don't support just v12. Maybe something like 'The required format of the sample sheets can be found in the SMRT Link User Guide' (https://www.pacb.com/support/documentation/)